ui/src/app/views/workspace-edit/component.css

```diff
@@ -12,3 +12,7 @@ label {
   width: 40em;
   padding: 1em;
 }
+
+.checkbox-group {
+  padding: 1em 0;
```

Comment:
Quick check: Does this make sense to scale with the font size of this element (`em`) over the base font size of the page (`rem`)?